app/MMB/src/appRegistry/ManageMyBookingPackage.js

```diff
@@ -11,5 +12,13 @@ type Props = {|
 |};
 
 export default class ManageMyBookingPackage extends React.Component<Props> {
-  render = () => <NavigationStack screenProps={this.props} />;
+  render = () => (
+    <MMBContext.Provider
+      accessToken={this.props.accessToken}
+      language={this.props.language}
+      currency={this.props.currency}
```

Comment:
Where is `language` and `currency` actually used? For example, the way how we should handle languages completely changed in our GraphQL. Are we preparing here for something that is maybe already invalid? 🙂

Reply:
currency is beeing used atleast here
![zep](https://cdn.zeplin.io/59a9818fe86dfd4f75f2c142/screens/68CC2E11-6F11-4C88-B8EF-AD44402D7965.png)

I will remove language, I guess you are right. It is not in use.